core/src/main/java/org/akaza/openclinica/domain/rule/action/NotificationActionProcessor.java

```diff
@@ -270,6 +270,10 @@ public void run() {
 
 		} else {
 			pDTO = buildNewPDTO();
+            message = message.replaceAll("\\\\n", "\n");
```

Comment:
While you are there can you please remove the empty System.out and format these lines.